### ISSUE TITLE
Eternal Safari problem ... let's summarize and solve it ;-)

### ISSUE THREAD

**Issue by ttimatkinss** [collab-project/videojs-record#534]
Facts:

1) VideoJs record demo does not work for Safari ... you can not play what you recorded (it's already sent as an issue in this forum)

2) Safari 14.3 will have MediaRecorder active as default, but as many people explained, we can not say users to activate a hidden Developer menu option in order to work with our application

3) Some other issues says the "technology" is blocked by RecordRTC (https://recordrtc.org/) ... a library that does not work in Safari either.

4) I've read all the issues here: https://www.google.com/search?q=videojs+record+safari

So it seems it's IMPOSSIBLE to Record a video in HTML5 from Safari (or iPhone) ...  hmmm ... sure?

Is there any REAL way to record a video (and play it or save it or send to server) from HTML5 working in Safari ?

THANK YOU !

**Comment by thijstriemstra** [comment]
I haven't personally looked at the new Safari yet. Would be good to add your summary to the existing safari ticket: #332 

Marking as duplicate of #332
